tests/test_integrated_channels/test_blackboard/test_client.py

```diff
@@ -20,16 +31,100 @@ class TestBlackboardApiClient(unittest.TestCase):
 
     def setUp(self):
         super(TestBlackboardApiClient, self).setUp()
+        self.token = "token"
         self.enterprise_config = BlackboardEnterpriseCustomerConfigurationFactory(
             client_id="id",
             client_secret="secret",
             blackboard_base_url="https://base.url",
-            refresh_token="token",
+            refresh_token=self.token,
         )
+        self.user_email = 'testemail@example.com'
+        self.course_id = 'course-edx+{}'.format(str(random.randint(100, 999)))
+        self.black_board_course_id = random.randint(100, 999)
```

Comment:
black_board is weird,, otherwise lgtm! good tests

Reply:
oh true. Consistency is best!!